incubator/hnc/pkg/stats/stats.go

```diff
@@ -57,9 +62,15 @@ func StartObjReconcile(gvk schema.GroupVersionKind) {
 	gk := gvk.GroupKind()
 	if _, ok := stats.objects[gk]; !ok {
 		stats.objects[gk] = &object{}
+		peakConcurrentObjectReconcile[gk] = 0
```

Comment:
Is this needed? I _think_ `m[k] = max(m[k], foo)` will work even if the key `k` doesn't exist yet, because `m[k]` will evaluate to the default value, which for an integer is zero. But I'm not certain.